client/landing/jetpack-cloud/components/log-item/docs/example.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+
+import React, { PureComponent, ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import LogItem from '../';
+import CardHeading from 'components/card-heading';
+
+export default class LogItemExample extends PureComponent {
```

Comment:
I wonder if this could just be a function instead of a `PureComponent`.

Reply:
Yes, but actually a functional component is [_slower_](https://medium.com/groww-engineering/stateless-component-vs-pure-component-d2af88a1200b) than Pure Components, so if I have a case like thing I stick to Pure :)